antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -497,7 +497,7 @@ def get(
 
         Returns: study data formatted in json
         """
-        self._safe_generation(metadata, timeout=60)
+        self._safe_generation(metadata, timeout=600)
```

Comment:
To be honest I am not sure why we have a timeout here.

If the generation task hangs forever we anyway have a problem.
If it's just "long", which may happen, the client will have its own timeout.

Reply:
Let's keep it for now .... it can have the small benefit of not keeping a fastapi thread busy for too long ...